migrateOldEpicGroups.js

```diff
@@ -0,0 +1,241 @@
+// Test
+// MongoClient.connect("mongodb://x:x@localhost:5555/epic", async function (err, client) {
+// Local
+MongoClient.connect("mongodb://localhost/epic", async function(err, client) {
+  if (!err) {
+    console.log("We are connected");
+    const db = client.db("epic");
+
+    // Get current organizations in database
+    var orgsData = await getOrgs(db);
+    console.log("orgs data:", orgsData.length);
+
+    // Get new organizations we want to import from JSON
+    let newOrgs = require(process.cwd() + "/newOrgs");
+    console.log("new orgs:", newOrgs.length);
+
+    // Get current contacts in database
+    let contactsData = await getContacts(db);
+    console.log("contacts data:", contactsData.length);
+
+    // Get new contacts we want to import from JSON
+    let newContacts = require(process.cwd() + "/newContacts");
+    console.log("new contacts:", newContacts.length);
+
+    let newGroups = require(process.cwd() + "/newGroups");
+    console.log("new groups:", newGroups.length);
+
+    // Importing in organizations
+
+    console.log("************* Processing Organizations *************");
+
+    for (let z = 0; z < newOrgs.length; z++) {
+      // log file to keep track of inserted items
+      var filedata = fs.readFileSync("doneOrgs.json", "utf8");
+
+      obj = JSON.parse(filedata); //now it's an object
+
+      var found = filedata.includes(newOrgs[z].name);
+
+      if (found) {
+        console.log("Already found, skipping", newOrgs[z].name);
+      } else {
+        console.log("Inserting", newOrgs[z].name);
+        insertOrg(db, newOrgs[z]);
+        obj.push({ id: "" + newOrgs[z]._id }, { name: "" + newOrgs[z].name });
+        json = JSON.stringify(obj); //convert it back to json
+        fs.writeFileSync("doneOrgs.json", json, "utf8"); // write it back
+      }
+    }
+
+    var all_orgs = await getOrgs(db);
+    console.log(
+      "\n\n\n********* Processed Organizations. Total Orgs is now ",
+      all_orgs.length
+    );
+
+    // Importing in contacts
+
+    console.log("\n\n\n************* Processing Contacts *************");
+
+    for (let z = 0; z < newContacts.length; z++) {
+      // log file to keep track of inserted items
+      var logContacts = fs.readFileSync("doneContacts.json", "utf8");
+
+      obj = JSON.parse(logContacts); //now it an object
+
+      var found = logContacts.includes(newContacts[z].displayName);
+
+      // Look up the organization document that matches the same orgName field for the Contact
+      orgLookUp = await findOrg(db, newContacts[z].orgName);
+
+      // Update Contact to link to organization's ObjectId
+      try {
+        orgID = orgLookUp[0]._id;
+        newContacts[z].org = orgID;
+      } catch (e) {
+        console.log("Error", e);
+        orgID = null;
+      }
+
+      newContacts[z].org = orgID;
+
+      // insert Contact to DB
+      if (found) {
+        console.log("Already found, skipping", newContacts[z].displayName);
```

Comment:
Let's comment out the log statements that log the contact names, just in case those somehow end up in elasticsearch.